<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbcFilter</class>
 <widget class="QFrame" name="DbcFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>27</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,1,0">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButtonDelete">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="gex.qrc">
          <normaloff>:/gex/icons/delete.ico</normaloff>:/gex/icons/delete.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelIndex">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string>index</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string> : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxParamNumber"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxOperator"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditParamValue"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAdd">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="gex.qrc">
          <normaloff>:/gex/icons/add.png</normaloff>:/gex/icons/add.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>10</width>
          <height>15</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
